### PR TITLE
SRCH-3046 Update README with standardized search-services language

### PR DESCRIPTION
**Pull request by danswick**
related: https://github.com/GSA/i14y/pull/193, https://github.com/GSA/search-services/pull/5

## Summary
 Updates README language to direct users to search-services for more information on setting up and running shared services.

### Checklist
Please ensure you have addressed all concerns below before marking a PR "ready for review" or before requesting a re-review:

#### Functionality Checks

- [x] Code is functional.

- [x] Automated checks pass, if applicable. If Code Climate checks do not pass, explain reason for failures: **N/A**: checks don't apply to README.

- [x] If your changes will be tested manually, you have run `bundle update` and committed your changes to Gemfile.lock. **N/A**: no manual tests required for README updates.
 
- [x] You have merged the latest changes from the target branch (usually `main`) into your branch.
 
- [x] If your target branch is NOT `main`, specify the reason:
 
- [x] PR title is either of the format **SRCH-#### \<description\>** matching the associated Jira ticket (i.e. "SRCH-123 implement feature X"), or **Release #.#.#** matching the release number
 
- [x] You have squashed your commits into a single commit (exception: your PR includes commits with formatting-only changes, such as required by Rubocop)

- [x] Your primary commit message is of the format **SRCH-#### \<description\>** matching the associated Jira ticket. **N/A**: I plan to use "squash and merge" instead of a manual squash.
 
#### Process Checks

- [x] You have specified an "Assignee", and if necessary, additional reviewers

### CODE REVIEW

**Comment by danswick** [comment]
@MothOnMars PR checklist has been updated!

File: README.md [hunk 1]
```diff
@@ -18,20 +18,32 @@ Use [RVM](https://rvm.io/) to install the version of Ruby specified in [.ruby-ve
...
 
-Some specs depend upon Elasticsearch having a valid trial license. A 30-day trial license is automatically applied when the cluster is initially created. If your license expires, you can rebuild the cluster by [rebuilding the container and its data volume](https://github.com/GSA/search-gov/wiki/Docker-Command-Reference/_edit#recreate-an-elasticsearch-cluster-useful-for-restarting-a-trial-license). 
-    
-    
-* [Kibana](https://www.elastic.co/kibana) - Kibana is not required, but can be very useful for debugging Elasticsearch. Confirm Kibana is available by visiting <http://localhost:5601>.
-
-* [MySQL](https://dev.mysql.com/doc/refman/5.7/en/) 5.7 - database, accessible from user 'root' with no password
-* [Redis](https://redis.io/) 5.0 - We're using the Redis key-value store for caching, queue workflow via Resque, and some analytics.
-* [Tika](https://tika.apache.org/) - for extracting plain text from PDFs, etc. The [Tika REST server](https://cwiki.apache.org/confluence/display/TIKA/TikaServer) runs on <http://localhost:9998/>.
+Verify that Elasticsearch 6.8.x is running on port the expected port (in this example, port 9200):
```
Comment (lorawoodford) [inline README.md:25]:
```suggestion
Verify that Elasticsearch 6.8.x is running on the expected port (port 9200 by default):
```

**Review by lorawoodford — CHANGES_REQUESTED** [review]
@danswick One tiny typo change/clarification!

**Review by lorawoodford — APPROVED** [review]
LGTM

**Pull request merged**